src/components/notifications/Onesender.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div class="mb-3">
+        <label for="host-onesender" class="form-label">{{ $t("HostOnesender") }}</label>
+        <input
+            id="host-onesender"
+            v-model="$parent.notification.onesenderURL"
+            type="url"
+            placeholder="https://xxxxxxxxxxx.com/api/v1/messages"
+            pattern="https?://.+"
+            class="form-control"
+            required
+        />
+    </div>
+
+    <div class="mb-3">
+        <label for="receiver-onesender" class="form-label">{{ $t("TokenOnesender") }}</label>
+        <input
+            id="receiver-onesender"
+            v-model="$parent.notification.onesenderToken"
+            type="text"
+            class="form-control"
+            required
+        />
+    </div>
+
+    <div class="mb-3">
+        <label for="webhook-request-body" class="form-label">{{ $t("TypeOnesender") }}</label>
+        <select
+            id="webhook-request-body"
+            v-model="$parent.notification.onesenderTypeReceiver"
+            class="form-select"
+            required
+        >
+            <option value="private">{{ $t("PrivateOnesender") }}</option>
+            <option value="group">{{ $t("GroupOnesender") }}</option>
+        </select>
```

Comment:
Could you please have a look if adding a helptext is benefitial for this option?
Selecting `private`/`group` in a selection field called `Group ID / Private` does not seem very obvious to endusers.
Feel free to overrule if it is better this way